### ISSUE TITLE
solr does not restart after a reboot

### ISSUE THREAD

**Issue by dtenenba** [Bioconductor/bioconductor.org_archive#14]
webadmin's crontab has this:

`@reboot cd $HOME/solr-5.2.1$ && bin/solr start > $HOME/solr.log 2>&1`

When the machine is rebooted, this job does not appear to run. There could be a few reasons why:
- Maybe this version of cron does not support `@reboot` jobs, but I thought all recent ones did.
- `solr/start` starts solr in the background, so even if this had worked, the relevant log messages
  would not appear in `$HOME/solr.log` as we seem to expect.

We do not want to reboot on production so maybe I will set up an AMI (not a docker container as it probably would not run crond on start) to test this.

@b-long

**Comment by dtenenba** [comment]
There is an obvious typo in that crontab entry, the $ sign after `cd $HOME/solr-5.2.1`. Testing to see if that is the issue. In the meantime, removed the typo on production as it is definitely wrong (whether the crontab will work now is a separate question).

**Comment by dtenenba** [comment]
OK, in my testing, removing that typo ($) did the trick. Rather than save the AMI, here is how I created it. I went to https://cloud-images.ubuntu.com/locator/ec2/ and picked an ubuntu 14.04 AMI in us-east-1 with an ebs instance type, launched it as t2.micro and ran:

```
sudo apt-get update
sudo apt-get install openjdk-7-jre-headless unzip
curl -LO http://apache.arvixe.com/lucene/solr/5.2.1/solr-5.2.1.tgz
tar zxf solr-5.2.1.tgz
```

Then I played around in the crontab. First I just tried to get ordinary jobs working (not `@reboot` jobs) and then after I had things the way I wanted, I modified the `@reboot` job. 
The solr process was running when I rebooted.

Another issue...`bin/solr start` starts solr in the background and output goes to a log file that I could not find. If I add the `-f` option the expected output goes to `$HOME/solr.log`. 

This works on my test instance and I've verified that it works on production as well. Modified crontab on production. Closing issue. @b-long
